composer/algorithms/mixup/mixup.py

```diff
@@ -19,75 +19,77 @@
+    defined by the ``mixing`` parameter. This parameter should be chosen
+    from a ``Beta(alpha, alpha)`` distribution for some parameter ``alpha > 0``.
+    Note that the same ``mixing`` is used for the whole batch.
 
     Args:
-        x (Tensor): input tensor of shape (B, d1, d2, ..., dn), B is batch size, d1-dn
-            are feature dimensions.
-        y (Tensor): target tensor of shape (B, f1, f2, ..., fm), B is batch size, f1-fn
-            are possible target dimensions.
-        n_classes: total number of classes.
-        interpolation_lambda (float, optional): coefficient used to interpolate between the
-            two examples. If provided, must be in ``[0, 1]``. If ``None``,
-            value is drawn from a ``Beta(alpha, alpha)`` distribution. Default: ``None``.
-        alpha (float, optional): parameter for the beta distribution over the
-            ``interpolation_lambda``. Only used if ``interpolation_lambda``
-            is not provided. Default: ``0.2``.
-        indices (Tensor, optional): Permutation of the batch indices `1..B`. Used
-            for permuting without randomness.
+        input (torch.Tensor): input tensor of shape ``(minibatch, ...)``, where
+            ``...`` indicates zero or more dimensions.
+        target (torch.Tensor): target tensor of shape ``(minibatch, ...)``, where
+            ``...`` indicates zero or more dimensions.
+        num_classes (int): total number of classes or output variables
+        mixing (float, optional): coefficient used to interpolate
+            between the two examples. If provided, must be in :math:`[0, 1]`.
+            If ``None``, value is drawn from a ``Beta(alpha, alpha)``
+            distribution. Default: ``None``.
+        alpha (float, optional): parameter for the Beta distribution over
+            ``mixing``. Ignored if ``mixing`` is provided. Default: ``0.2``.
+        indices (Tensor, optional): Permutation of the samples to use.
+            Default: ``None``.
 
     Returns:
-        x_mix (Tensor): batch of inputs after mixup has been applied
-        y_mix (Tensor): labels after mixup has been applied
-        perm (Tuple[Tensor]): the permutation used
+        input_mixed (torch.Tensor): batch of inputs after mixup has been applied
+        target_mixed (torch.Tensor): labels after mixup has been applied
+        perm (torch.Tensor): the permutation used
+
+    Example:
+        .. testcode::
+
+            import torch
+            from composer.functional import mixup_batch
+
+            N, C, H, W = 2, 3, 4, 5
+            num_classes = 10
+            X = torch.randn(N, C, H, W)
+            y = torch.randint(num_classes, size=(N,))
+            X_mixed, y_mixed, perm = mixup_batch(
+                X, y, num_classes=num_classes, alpha=0.2)
     """
-    if interpolation_lambda is None:
-        interpolation_lambda = _gen_interpolation_lambda(alpha)
+    if mixing is None:
+        mixing = _gen_interpolation_lambda(alpha)
```

Comment:
Should this call something like `_gen_mixing(alpha)` instead?

Reply:
I'm happy to go with whatever you think makes sense. I'm was hoping to mirror the notation from the paper internally for all the reasons you identified, but if you think having the differing names is net negative that works too.

I'll go with this change unless you tell me otherwise.